internal/project/project.go

```diff
@@ -326,6 +344,75 @@ func (p *Project) addRoot(info *ScriptInfo) {
 	p.markAsDirty()
 }
 
+func (p *Project) LoadConfig() error {
+	if p.kind != KindConfigured {
+		panic("loadConfig called on non-configured project")
+	}
+
+	if configFileContent, ok := p.host.FS().ReadFile(p.configFileName); ok {
+		configDir := tspath.GetDirectoryPath(p.configFileName)
+		tsConfigSourceFile := tsoptions.NewTsconfigSourceFileFromFilePath(p.configFileName, p.configFilePath, configFileContent)
+		parsedCommandLine := tsoptions.ParseJsonSourceFileConfigFileContent(
+			tsConfigSourceFile,
+			p.host,
+			configDir,
+			nil, /*existingOptions*/
+			p.configFileName,
+			nil, /*resolutionStack*/
+			nil, /*extraFileExtensions*/
+			nil, /*extendedConfigCache*/
+		)
+
+		p.logf("Config: %s : %s",
```

Comment:
I do wonder if we should be making some of this lazy or something, given we're unconditionally converting stuff to json to log it and such.

Reply:
Ah, this is code move, so not new exactly anyhow. Certainly not this PR or anything.